### ISSUE TITLE
Error when disconnecting WalletConnect

### ISSUE THREAD

**Issue by PhilGabardo** [TxnLab/use-wallet#21]
I am seeing the following error when attempting to disconnect a Pera client:

```
Unhandled Runtime Error
TypeError: Cannot read private member #client from an object whose class did not declare it

Call Stack
Object.disconnect [as callback]
node_modules/@txnlab/use-wallet/dist/esm/index.js (1127:0)
eval
node_modules/@walletconnect/core/dist/esm/events.js (41:0)
Array.forEach
<anonymous>
EventManager.trigger
node_modules/@walletconnect/core/dist/esm/events.js (35:0)
WalletConnect._handleSessionDisconnect
node_modules/@walletconnect/core/dist/esm/index.js (633:0)
WalletConnect.killSession
node_modules/@walletconnect/core/dist/esm/index.js (418:0)
```

The error only appears to happen when the webpage is reloaded after the client is set as active.

**Comment by gabrielkuettel** [comment]
Thanks for reporting this error! Should be fixed in v1.0.3.

https://github.com/TxnLab/use-wallet/releases/tag/v1.0.3

**Comment by PhilGabardo** [comment]
thanks for the quick fix :)
